.luacheckrc

```diff
@@ -86,6 +87,8 @@ files["**/*.lua"] = {
         "ReturnValuesFn",
         "RPC",
         "SendRPCToServer",
+        "ThePlayer",
```

Comment:
It's better to sort these alphabetically (case insensitive) to find stuff easier.